flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -193,7 +195,8 @@ aileron_down_limit = 17 ; Aileron max deflection down angle (absolute value) (DE
 rudder_limit = 25 ; Rudder max deflection angle (absolute value) (DEGREES)
 rudder_trim_limit = 20 ; Rudder trim max deflection angle (absolute value) (DEGREES)
 elevator_trim_limit = 13.5 ; Elevator trim max angle (absolute value) (DEGREES)
-spoiler_limit = 35 ; Spoiler max deflection angle (absolute value) (DEGREES)
+spoiler_limit = 35 ; Spoiler max deflection angle when in the air (absolute value) (DEGREES)
+air_spoiler_limit = 35 ; Spoiler max deflection angle when on the ground (absolute value) (DEGREES)
```

Comment:
The descriptions look mixed up, ```spoiler_limit``` is "Spoiler max deflection angle, in degrees (absolute values only) when on the ground." and vice-versa

Reply:
Oops, good catch!  Thanks!